cmd/diff.go

```diff
@@ -44,6 +44,9 @@ var diffCmd = &cobra.Command{
 	Use:   "diff [<env1> [<env2>]] [-f <file-or-dir>]",
 	Short: "Display differences between server and local config, or server and server config",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Usage()
```

Comment:
Can we display an error here that explains what went wrong instead of outputting usage?